src/fides/api/ops/service/messaging/message_dispatch_service.py

```diff
@@ -305,10 +315,12 @@ def _twilio_sms_dispatcher(
     try:
         if messaging_service_id:
             client.messages.create(
-                to=to, messaging_service_sid=messaging_service_id, body=message
+                to=to.strip(), messaging_service_sid=messaging_service_id, body=message
```

Comment:
what other processing / cleaning might we need from the identity input phone number? Should we expect dashes `-` might appear too? Need to research what validation we have on phone number identity

Reply:
will identity phone number always include `+` at the beginning?

Reply:
Should we normalize the phone numbers into [E.164](https://en.wikipedia.org/wiki/E.164) when we initially collect the phone number? Then we can manipulate it as needed before sending it to Twilio.

Reply:
Issue created here- https://github.com/ethyca/fides/issues/1730